INSS.FIP.Web/ViewModels/SearchParametersViewModel.cs

```diff
@@ -8,6 +8,7 @@ namespace INSS.FIP.Web.ViewModels;
 public class SearchParametersViewModel : IValidatableObject
 {
     private const string RegExForName = "^[a-zA-Z ]+(([',.\\-][a-zA-Z ])?[a-zA-Z ]*)*$";
+    private const string RegExForCompanyName = "^[a-zA-Z ]+(([',&.\\-][a-zA-Z ])?[a-zA-Z ]*)*$";
```

Comment:
The ticket says , allow 100 characters including & and £ , Cant see £ in the RegEx

Reply:
Refer the updated ticket ' £' is not required. and yes characters length could be of max 100. but this is already in place so don't require any code changes.